pkg/controllers/node/termination/terminator/eviction.go

```diff
@@ -90,10 +92,15 @@ type Queue struct {
 
 func NewQueue(kubeClient client.Client, recorder events.Recorder) *Queue {
 	queue := &Queue{
-		RateLimitingInterface: workqueue.NewRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(evictionQueueBaseDelay, evictionQueueMaxDelay)),
-		set:                   sets.New[QueueKey](),
-		kubeClient:            kubeClient,
-		recorder:              recorder,
+		RateLimitingInterface: workqueue.NewRateLimitingQueueWithConfig(
+			workqueue.NewItemExponentialFailureRateLimiter(evictionQueueBaseDelay, evictionQueueMaxDelay),
+			workqueue.RateLimitingQueueConfig{
+				Name:            "evictionQueue",
```

Comment:
Same comment here about naming. Can we make a convention for both the queue and the reconciler so that we get consistent naming